test/bigtest/tests/line-edit-test.js

```diff
@@ -195,4 +195,29 @@ describe('Line edit test', () => {
       });
     });
   });
+
+  describe('Check existing Item Details Title positive case', () => {
+    const EMPTY_STRING = '';
+
+    beforeEach(async function () {
+      await lineEditPage.itemDetailsAccordion.inputTitle('TEST_VALUE');
+      await lineEditPage.updateLineButton.click();
+    });
+
+    it('Shouldn\'t provide any warning message', () => {
```

Comment:
hmm, guys, I was thinking that if we don't use `should` then the proper names could be like `it('doesn't provide any warnings')`

Reply:
fixed